lib/parser.js

```diff
@@ -35,6 +35,41 @@ const blockOption = seq(
   }
 );
 const blockOptions = blockOption.many();
+
+//block
+const flagBegin = lexeme(string('#+BEGIN_'));
+const flagEnd= lexeme(string('#+END_'));
+const blockName = lexeme(regex(/[^\s]+/i));
+const content = lexeme(regex(/[^#+END_]*/));
+const block = seq(
+  startOfLine, flagBegin, blockName, singleLineText, content , flagEnd, singleLineText
+).map(
+  (val) => {
+    let ctt = val[4] ? val[4].split("\n") : 'None';
```

Comment:
What's the `ctt` for? @b232wang please use more meaningful word to name variable :-)